tests/e2e/gc_metadata_syncer.go

```diff
@@ -842,7 +842,7 @@ var _ = ginkgo.Describe("[csi-guest] pvCSI metadata syncer tests", func() {
 	// 9. Wait for Pod name to be deleted in CNS.
 	// 10. Delete PVCs.
 
-	ginkgo.It("Multiple PVCs - Verify Pod Name is updated/deleted on CNS", func() {
+	ginkgo.It("Multiple PVCs - Verify Pod Name is updated/deleted on CNS", ginkgo.Label(p0, block, tkg, vc70), func() {
```

Comment:
Can be P1.  This is to validate same update operation where POD  is attached with multiple PVC's